cabal.project

```diff
@@ -1,12 +1,19 @@
--- See also http://cabal.readthedocs.io/en/latest/nix-local-build-overview.html
-
 -- NB: ./quickcheck is deprecated
-packages: ./core
-          ./hunit
-          ./quickcheck2
-          ./example
+packages:
+  ./core
+  ./hunit
+  ./quickcheck2
+  ./example
 
 -- On Debian/Ubuntu you may need the following to workaround an issue in libxml's packaging
 
 -- package libxml
 --     extra-include-dirs: /usr/include/libxml2
+
+package *
```

Comment:
@andreasabel interesting!  Does this now only apply to local packages, or to the whole dependency tree?  I think at least historically, there was some issue here.

What I tend to use by now is:

```cabal
program-options
  ghc-options: ...
```
This only applies to local packages.

Reply:
Ah thanks, didn't know this trick yet!